src/entities/Badges/utils.ts

```diff
@@ -124,3 +111,24 @@ export async function createSpec(badgeCid: string) {
   logger.log('Create badge spec with txn hash:', txn.hash)
   return txn.hash
 }
+
+// TODO: separate utils for DAO badges from utils for contract interactions
+
+export async function getLandOwnerAddresses(): Promise<string[]> {
+  const LAND_API_URL = 'https://api.decentraland.org/v2/tiles?include=owner&type=owned'
+  type LandOwner = { owner: string }
+  try {
+    const response: ApiResponse<{ [coordinates: string]: LandOwner }> = await (await fetch(LAND_API_URL)).json()
```

Comment:
Not related with this PR but I think we can refactor and use `Land.getTiles()` here to simplify this. @ncomerci